### PR TITLE
update ssh with fido2 guide

### PR DESCRIPTION
**Pull request by elukewalker**

### CODE REVIEW

File: content/SSH/Securing_SSH_with_FIDO2.adoc [hunk 1]
```diff
@@ -1,173 +1,254 @@
...
-Host example.demo.com 
-IdentityFile ~/.ssh/id_ecdsa_sk_rk_credential_example_com
+# Support public key cryptography (includes FIDO2)
+PubkeyAuthentication yes
+# Enforce User Verification
+PubkeyAuthOptions verify-required
+# Public keys location
+AuthorizedKeysFile .ssh/authorized_keys
+# Allow root only with MFA
+PermitRootLogin without-password
```
Comment (joostd) [inline content/SSH/Securing_SSH_with_FIDO2.adoc:139]: I think `without-password` is deprecated in favour of `prohibit-password`

File: content/SSH/Securing_SSH_with_FIDO2.adoc [hunk 2]
```diff
@@ -1,173 +1,254 @@
...
++
+. Paste the public key and save.
 
-To use the `ed25519` curve (requires a YubiKey with firmware 5.2.3 or higher), use the following command instead:
-`ssh-keygen -t ed25519-sk`
---
+*Using SSSD (Optional)*
 
-. SSH will request the user to enter their PIN and touch the device. When this request occurs, the YubiKey will flash.
+* For centralized management, store public keys in LDAP (Active Directory) using SSSD.
```
Comment (joostd) [inline content/SSH/Securing_SSH_with_FIDO2.adoc:171]: Perhaps change to 
    (for instance Active Directory)

File: content/SSH/Securing_SSH_with_FIDO2.adoc [hunk 3]
```diff
@@ -1,173 +1,254 @@
...
+[source,sh]
+----
+journalctl -r /usr/sbin/sshd
+----
++
+* Run SSH in debug mode:
++
+[source,sh]
+----
+ssh -vvvv user@host
```
Comment (joostd) [inline content/SSH/Securing_SSH_with_FIDO2.adoc:217]: I think the maximum is `-vvv`

File: content/SSH/Securing_SSH_with_FIDO2.adoc [hunk 4]
```diff
@@ -1,173 +1,254 @@
...
--- 
-`export SSH_SK_HELPER="/mnt/c/Program Files/OpenSSH/ssh-sk-helper.exe"`
---
+[source,sh]
+----
+chmod 600 ~/.ssh/id_ed25519_sk
+----
 
-. The SSH identity files (id_ecdsa_sk and  id_ecdsa_sk.pub) must be available in `~/.ssh/`
+*Verify the FIDO2 credential on security key*
```
Comment (joostd) [inline content/SSH/Securing_SSH_with_FIDO2.adoc:229]: This is more useful when compared to the credential id in the private key file, but that requires additional tooling I think?

**Review by joostd — CHANGES_REQUESTED** [review]
Please see comments if you want to change anything. Otherwise a much clearer overview, so great improvement over the original.

File: content/SSH/Securing_SSH_with_FIDO2.adoc (hunk 4 above)
Comment (elukewalker) [reply content/SSH/Securing_SSH_with_FIDO2.adoc:229]: Is the recommendation to remove this section?

**Review by joostd — APPROVED** [review]
LGTM!

**Pull request merged**